p2p/transport/websocket/listener.go

```diff
@@ -109,8 +110,12 @@ func (l *listener) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Set an arbitrarily large read limit since we don't actually want to limit the message size here.
+	c.SetReadLimit(math.MaxInt64 - 1) // -1 because the library adds a byte for the fin frame
```

Comment:
```suggestion
	// Set an arbitrarily large read limit since we don't actually want to limit the message size here.
	// See https://github.com/nhooyr/websocket/issues/382 for details.
	c.SetReadLimit(math.MaxInt64 - 1) // -1 because the library adds a byte for the fin frame
```